tools/vagrant/provision-vagrant.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
Any chance this script - which duplicates most of the stuff used in the docker build image - could be moved to tools/ and maybe merged with one of the 2-3 other scripts doing the same ? Not a huge problem, it may be better not to complicate things too much - but things will get out of sync.